docs/develop/sdks-integrations/deploying-javascript-web-sdk.md

```diff
@@ -0,0 +1,89 @@
+---
+
+# Momento Web SDK for Javascript in browsers
+
+Momento provides two JavaScript SDKs; one for Node.js and one for other web applications.  The two SDKs have
+identical APIs, so your code will look the same except for `import` statements, but under the hood they are
+built for optimal performance and compatibility in different JavaScript runtime environments.
+
+The Node.js SDK is best suited for server-side use cases. The Momento Web SDK, however, allows developers to write
+JavaScript code that runs in a browser and communicates directly with Momento services. This allows you to avoid the
+typical overhead of building and operating your own web service to mediate cache or pub/sub calls between the browser
+and Momento.  It also means one less hop for your web traffic, so you can get even performance out of your browser
+application.  The best of both worlds1
+
+You can also use the Web SDK in other non-Node.js JavaScript environments, such as CloudFlare.
+
+## Momento Web SDK and Momento Topics
+
+Momento Topics significantly simplifies publisher-subscriber communication in a browser. Traditionally, developers
+must build out server-side infrastructure to manage web socket connections for all active browser clients. A classic
+example of this is a chat application embedded in a website; you are not only building the client code for the browser,
+but the server-side code for routing all the communications.
+
+This server-side complexity is eliminated by incorporating Momento Topics with the Momento Web SDK. Developers can
+subscribe to Momento Topics directly from the browser.  Momento then takes care of all communication when messages are
+published to the topic, eliminating the need for custom server-side infrastructure for web sockets!
+
+# Using the Web SDK for browsers
+
+While the API calls are [identical to the Momento node.js SDK](/develop/guides/cheat-sheets/momento-cache-nodejs-cheat-sheet.md),
+the import/require statement will consume the `@gomomento/sdk-web` package from npm, instead of `@gomomento/sdk` (which is
+the Node.js SDK).
+
+Here's an example import statement for the Web SDK:
+
+`import {CacheClient} from ‘@gomomento/sdk-web’;`
+
+# Links
+
+Here are some other useful links for getting started with the Momento Web SDK:
+
+* [Example Chat App](https://github.com/momentohq/example-chat-app) - fully functional chat application built using the Web SDK!
+
+![chat screenshot](./images/web-sdk-chat-app.png)
+
+* [Momento Node.js Cheat Sheet](/develop/guides/cheat-sheets/momento-cache-nodejs-cheat-sheet.md)
+* [Example Web SDK scripts on github](https://github.com/momentohq/client-sdk-javascript/examples/web)
+
+# Credentials for Browsers
+
+In order for your browser application to communicate with Momento, you will need to provide the browser with a Momento
+auth token. The recommended practice is to generate a single-use Momento auth token that has expiring credentials so that
```

Comment:
We're calling these API tokens instead of auth tokens, right?

Reply:
No, we went back to auth tokens as to switch to API tokens would have required changes in 100s of places in docs, code repos, existing SDKs, etc.

Reply:
nope, we decided to fully lean into the term "auth token" because it already appeared a bunch of places in our docs and APIs